packs/prometheus/config.yml

```diff
@@ -1,19 +1,39 @@
 id: 0dcf38e5-0e99-499b-94ca-2d515a9b1df1
 name: prometheus
 description: |
-  # What is Prometheus?
-  
-  Prometheus is a free software application used for event monitoring and alerting.
+   ## Prometheus Monitoring Dashboard - The Complete Quickstart
 
-  ## Get started!
+  Prometheus is an open-source monitoring and alerting toolkit. Developers often use it to monitor Kubernetes, an open-source environment used for automating application deployment, scaling, and containerized applications management. 
 
-  New Relic offers 2 types of Prometheus integrations:
+  Setting up Prometheus is straightforward, but scaling up and managing is not. That’s where New Relic steps in.
+
+  ### New Relic's Prometheus Quickstart
+
+  New Relic offers two Prometheus integration schemes, Remote Write and OpenMetrics. 
+
+  Remote Write is ideal for well-established Prometheus infrastructures. It provides easy access to your metrics and only takes one line of yaml in your configuration for access. 
+
+  OpenMetrics supports both Docker and Kubernetes. It allows for more visibility across multiple container platforms. We recommend OpenMetrics for clients who don’t want to manage Prometheus servers and local storage themselves. Getting started is fairly easy, and once the integration is set up, you can query data on memory usage for pods in deployment, separate various metrics into min, max, and sum, and view raw metric values all in one place.
+
+  New Relic's Prometheus Integration stores all kinds of telemetry data – whether open-source, vendor-specific, or vendor-agnostic. 
+
+  ### Value of the Prometheus Pack 
+
+  New Relic’s quickstart features a myriad of options to make DevOps easier. Although there are many ways to use Prometheus data in New Relic, we’ll break these down into OpenMetrics and Remote Write to help you decide on the best option for you:
+
+  #### OpenMetrics:
+  - Build customized dashboards to monitor Kubernetes HPA capacity
```

Comment:
Kubernetes HPA and Node readiness are good examples of OpenMetrics data.
But there is a lot more, should we add a more generic statement here that these are just examples and any type of OpenMetrics endpoint can be instrumented?